cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,21 @@
 /// <reference types='cypress' />
+import { LaptopsTab } from '../support/pages/customerUserWay';
 
-describe('', () => {
-  before(() => {
-
+const Laptops = new LaptopsTab();
+describe('Userflow using POM', () => {
+  before('open webpage', () => {
+    Laptops.navigateToHome();
   });
 
-  it('', () => {
-
+  it('full userway', () => {
+    Laptops.clickLaptopTab();
+    Laptops.clickSonyVaio7();
+    Laptops.addToTheCart();
+    Laptops.clickCartLink();
+    Laptops.checkItemInTheCart();
+    Laptops.clickThePlaceOrderButton();
+    Laptops.filoutForm();
```

Comment:
fillOutForm
I would like to see Laptops page object, not sure if this is related to Laptops